src/libraries/System.Private.CoreLib/src/System/IO/FileStream.cs

```diff
@@ -130,6 +130,43 @@ public FileStream(string path, FileMode mode, FileAccess access, FileShare share
         }
 
         public FileStream(string path, FileMode mode, FileAccess access, FileShare share, int bufferSize, FileOptions options)
+            : this(path, mode, access, share, bufferSize, options, 0)
+        {
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="T:System.IO.FileStream" /> class with the specified path, creation mode, read/write and sharing permission, the access other FileStreams can have to the same file, the buffer size,  additional file options and the allocation size.
+        /// </summary>
+        /// <param name="path">A relative or absolute path for the file that the current <see langword="FileStream" /> object will encapsulate.</param>
+        /// <param name="mode">One of the enumeration values that determines how to open or create the file.</param>
+        /// <param name="access">A bitwise combination of the enumeration values that determines how the file can be accessed by the <see langword="FileStream" /> object. This also determines the values returned by the <see cref="P:System.IO.FileStream.CanRead" /> and <see cref="P:System.IO.FileStream.CanWrite" /> properties of the <see langword="FileStream" /> object. <see cref="P:System.IO.FileStream.CanSeek" /> is <see langword="true" /> if <paramref name="path" /> specifies a disk file.</param>
+        /// <param name="share">A bitwise combination of the enumeration values that determines how the file will be shared by processes. The default value is Read.</param>
+        /// <param name="bufferSize">A positive <see cref="T:System.Int32" /> value greater than 0 indicating the buffer size. The default buffer size is 4096.</param>
```

Comment:
You got a build error here for using a the full type name for a primitive type. Need to change this to:
```suggestion
        /// <param name="bufferSize">A positive <see cref="int" /> value greater than 0 indicating the buffer size. The default buffer size is 4096.</param>
```

Reply:
> You got a build error here for using a the full type name for a primitive type

I did not get any build errors and the CI has passed. Does it mean that it would be a build error in the docs repo?

Reply:
It recently happened in another PR, but the code was not living in System.Private.CoreLib, it had its own solution and project. I wonder if that's making a difference. I'll check that later. Nothing to worry about here.